data-otservbr-global/scripts/creaturescripts/quests/grave_danger/zelosDeath.lua

```diff
@@ -0,0 +1,28 @@
+local config = {
+    centerPosition = Position(33443, 31545, 13),
+	rangeX = 11,
+	rangeY = 11,
+}
+
+local KingzelosDeath = CreatureEvent("zelosDeath")
+local removeMonsters = false
+
+function KingzelosDeath.onPrepareDeath(creature)
+    local spectators = Game.getSpectators(config.centerPosition, false, false, config.rangeX, config.rangeX, config.rangeY, config.rangeY)
+    for _, cid in pairs(spectators) do
+        if removeMonsters and cid:isMonster() and not cid:getMaster() then
```

Comment:
How does it work? This code will never be executed since removeMonsters is false and is never modified (is it supposed to be manually modified?)